src/CBT.After.Any.Targets/CBT.After.Any.Targets.nuproj

```diff
@@ -15,10 +15,7 @@
   </ItemGroup>
   <ItemGroup>
    <Dependency Include="CBT.Microsoft.Common.targets">
-      <Version>1.0.0</Version>
```

Comment:
as discussed removing the reference to CBT.Traversal but not deprecating the package just yet.  This is so it isn't auto pulled into build and enabled.